src/libaktualizr/primary/aktualizr_test.cc

```diff
@@ -1908,6 +1926,7 @@ int main(int argc, char** argv) {
   logger_init();
   logger_set_threshold(boost::log::trivial::trace);
 
+  meta_temp = new MetaFakeInTemp;
```

Comment:
We avoid creating objects with `new` in general.

It's not needed in this case, you can just declare `TemporaryDirectory dir;`  and `MetaFake meta_fake;` in main (or grouped in an anonymous struct if you prefer) + copy the path to a global `boost::filesystem`.

Everything will stay in scope while the tests run.